### PR TITLE
fix: InputGroup now can accept custom classNames

### PR DESCRIPTION
**Pull request by leannerivera**
### Description

InputGroups now accept custom classNames

fixes:  #408

### CODE REVIEW

**Comment by leannerivera** [comment]
Done. :)

File: src/InputGroup/InputGroup.js [hunk 1]
```diff
@@ -87,6 +89,7 @@ export class InputGroup extends Component {
                 );
 
                 const inputNumberClasses = classnames(
+                    className,
```
Comment (jbadan) [inline src/InputGroup/InputGroup.js]: It looks like this should be a different prop for the `input` className versus the `div` className. Maybe we should have `className` that get applied to the parent `div` and an `inputClassName` that gets applied to the `input` element.

**Comment by jbadan** [comment]
The new props should also be added to the PropTypes list at the bottom of the file :)

**Comment by leannerivera** [comment]
> The new props should also be added to the PropTypes list at the bottom of the file :)

Do you mean like the changes made in current push?

File: src/InputGroup/InputGroup.js [hunk 2]
```diff
@@ -59,6 +59,8 @@ export class InputGroup extends Component {
 
     render() {
         const {
+            inputClassName,
+            inputType,
```
Comment (jbadan) [inline src/InputGroup/InputGroup.js]: I'm getting duplicate declaration errors on this - I think `inputType` is also on line 74

File: src/InputGroup/InputGroup.js [hunk 3]
```diff
@@ -280,6 +299,7 @@ InputGroup.propDescriptions = {
     addon: 'The value of the add-on.',
     addonPos: 'Location of the add-on relative to the input.',
     inputId: 'Value for the `id` attribute on the `<input>` element.',
+    inputClassName: 'Value for the `classname` attribute on the `<input>` element.',
```
Comment (jbadan) [inline src/InputGroup/InputGroup.js]: Let's copy the current default for classNames - "CSS class(es) to add to the `<input>` element."

**Comment by jbadan** [comment]
This is looking great! I think once we address those two outstanding issues this will be ready to ship! 🎉

File: src/InputGroup/InputGroup.js (hunk 2 above)
Comment (leannerivera) [reply src/InputGroup/InputGroup.js]: oops looks like I merged this wrong. should be fixed with latest push.

**Comment by greg-a-smith** [comment]
@cjskillingstad is correct.  We are using npm 6.4.1 which is packaged with node 10.15.0.  I also agree that we shouldn't be committing any changes to `package-lock.json` if there are no changes to `package.json`.

@leannerivera Have you setup `nvm` (Node Version Manager) yet?  It's a really nice way of switching Node and NPM versions quickly (since we use different versions on different repos).

**Comment by leannerivera** [comment]
@greg-a-smith @cjskillingstad my mistake. haven't done nvm but will do.

**Pull request merged**